packages/redhat/common/metalk8s-sosreport.spec

```diff
@@ -0,0 +1,43 @@
+Name:		metalk8s-sosreport
+Version:	_VERSION_
+Release:	2%{?dist}
+Summary: 	Metalk8s SOS report custom plugins
+
+BuildRequires: /usr/bin/pathfix.py
+
+Requires: python3 >= 3.6
+Requires: sos >= 3.1
+# NameError on FileNotFoundError in sos 3.5 python2.7
+Conflicts: sos = 3.5
+
+ExclusiveArch:  x86_64
+ExclusiveOS:    Linux
+
+License:        GPL+
```

Comment:
Is it?

Reply:
I'm not really familiar with licensing stuff and it's not related to that PR, but as far as I understand, it is.

sosreport report is distributed under GNU GPL license which is a copyleft license:
https://github.com/sosreport/sos/blob/master/LICENSE

```
The GPL series are all copyleft licenses, which means that any derivative work must be distributed under the same or equivalent license terms.
```
source: https://en.wikipedia.org/wiki/GNU_General_Public_License

If it's not, could you explain why ?

Reply:
`sosreport`/`sos` is GPL. Our plugin is likely not a 'derived work', and all code in MetalK8s is APL2-licensed. So I'd argue our SOS plugin should also be APL2 licensed?